README.md

```diff
@@ -167,16 +167,17 @@ apply-manifests     10 seconds ago
 update-deployment   4 seconds ago
 ```
 
-We will be using `buildah` clusterTasks, which gets installed along with Operator. Operator installs few ClusterTask which you can see.
+We will be using `buildah` task, which gets installed along with Operator. Operator installs few tasks in namespace `openshift-pipelines` which you can see.
 
 ```bash
-$ tkn clustertasks ls
-NAME                       DESCRIPTION   AGE
-buildah                                  1 day ago
-buildah-v0-14-3                          1 day ago
-git-clone                                1 day ago
-s2i-php                                  1 day ago
-tkn                                      1 day ago
+$ tkn tasks ls -n openshift-pipelines
+NAME                      DESCRIPTION              AGE
+buildah                   Buildah task builds...   1 day ago
+git-cli                   This task can be us...   1 day ago
+git-clone                 This object represe...   1 day ago
+hello                                              1 day ago
```

Comment:
what is this `hello` task  ??

Reply:
My bad, I was testing something on my cluster. Fixed

Reply:
no worries 
Thank you